src/rantanplan/syllabification.py

```diff
@@ -50,7 +50,7 @@
 SPACE = "SPACE"
 STRONG_VOWELS = set("aeoáéóÁÉÓAEO")
 WEAK_VOWELS = set("iuüíúIÍUÜÚ")
-LIAISON_FIRST_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚyY")
+LIAISON_FIRST_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚyYhH")
```

Comment:
This is to mark synalephas for cases such as "oh abeja"

Reply:
:+1: